src/main/java/net/minecraftforge/fluids/BlockFluidBase.java

```diff
@@ -150,7 +152,7 @@ public BlockFluidBase(Fluid fluid, Material material)
         this.setTickRandomly(true);
         this.disableStats();
 
-        this.fluidName = fluid.getName();
+        this.fluidName = fluid.getRegistryName();
```

Comment:
Add `com.google.common.base.Preconditions#checkNotNull(fluid.getRegistryName())` at the top of this constructor because it's very easy to forget to set the registry name before calling this. It wasn't so easy to have a null `name` before because it was in the fluid constructor. This will help people port from the old registry to the new one more easily.